src/main/java/games/strategy/triplea/delegate/MustFightBattle.java

```diff
@@ -2780,4 +2789,28 @@ public void unitsLostInPrecedingBattle(final IBattle battle, final Collection<Un
   private static Map<Unit, Collection<Unit>> transporting(final Collection<Unit> units) {
     return TransportTracker.transporting(units);
   }
+
+  /**
+   *  Return attacking from Collection.
+   */
```

Comment:
It would be more appropriate (and DRY) to place the Javadoc on the `abstract` method in the superclass.  The Javadoc tool will automatically inherit the comment from an `abstract`/interface method onto an `@Override`.  In addition, the Google Style Checkstyle ruleset should not require comments on `@Override`s--only on the originating method (where applicable).

(This comment applies to `getAttackingFrom()`, `getAttackingFromMap()`, and `getAmphibiousAttackTerritories()` in `MustFightBattle` and `NonFightingBattle`.)